config/locales/en.yml

```diff
@@ -620,6 +629,21 @@ en:
       passed_html: "<a href=\"https://access-your-teaching-qualifications.education.gov.uk/qualifications/start\" class=\"govuk-link\">Access your teaching qualifications</a> to get your certificate."
       failed: "If you think this is mistake or need to appeal this outcome, contact your training provider."
 
+  npq_separation:
+    admin:
+      finance:
+        statements:
+          payment_authorisations:
+            new:
+              title: "Check %{statement_name} statement details before authorising for payment"
+              button: "Authorise for payment"
+              checks_done_legend: Have all necessary assurance checks been done?
+            # tag:
```

Comment:
Left in because they'll be needed for the follow on PR for integration with CPDNPQ-2141